src/Microsoft.AspNetCore.Mvc.Abstractions/ModelBinding/ModelBindingContext.cs

```diff
@@ -232,69 +82,93 @@ public ModelStateDictionary ModelState
         /// <remarks>
         /// The <see cref="ModelMetadata"/> property must be set to access this property.
         /// </remarks>
-        public Type ModelType => ModelMetadata?.ModelType;
+        public abstract Type ModelType { get; }
 
         /// <summary>
-        /// Gets or sets a model name which is explicitly set using an <see cref="IModelNameProvider"/>.
-        /// <see cref="Model"/>.
+        /// Represents the <see cref="ModelBinding.OperationBindingContext"/> associated with this context.
```

Comment:
I didn't write the text, just moved it from one class to another, would prefer to defer the copy edits for an API update sweep.